homeassistant/components/unifi/config_flow.py

```diff
@@ -41,6 +45,13 @@
 DEFAULT_SITE_ID = "default"
 DEFAULT_VERIFY_SSL = False
 
+_LOGGER = logging.getLogger(__name__)
+
+MODEL_PORTS = {
```

Comment:
Since the unifi cloud key is now also on UniFi OS can that also be discovered?

Reply:
It doesn't seem to be available via SSDP